### ISSUE TITLE
Cluster unreachable after migrate-domain (tls: internal error)

### ISSUE THREAD

**Issue by ncantelmo** [flynn/flynn#3801]
Running `migrate-domain` on an existing cluster can lead to the cluster becoming (seemingly) unreachable. Some context for this issue:

- new-domain.io did not yet have an SSL certificate
- the cluster install process (~2 weeks old) was still running in another terminal window
- the cluster was on DigitalOcean, 3x 2 GB / 40 GB instances, no external blobstore

```shell
$ flynn route add http new-domain.io
http/new-route-id

$ flynn route
ROUTE                          SERVICE     ID                  STICKY  LEADER  PATH
http:new-domain.io             my-app-web  http/new-route-id   false   false   /
http:my-app.xxxx.flynnhub.com  my-app-web  http/orig-route-id  false   false   /

$ flynn cluster migrate-domain new-domain.io
Migrating cluster domain (this can take up to 2m0s)...
Changed cluster domain from "xxxx.flynnhub.com" to "new-domain.io"
Updated local CLI configuration

$ flynn route
no apps in git remotes

$ git remote set-url flynn https://new-domain.io/my-app.git
$ flynn env
no apps in git remotes

$ git remote set-url flynn https://git.new-domain.io/my-app.git
$ flynn env
Get https://controller.new-domain.io/apps/my-app/release: remote error: tls: internal error
$ flynn log
Get https://controller.new-domain.io/apps/my-app/log?stream_types=stdout%2Cstderr: remote error: tls: internal error
```

**Comment by KidkArolis** [comment]
The **Certificates** section of this blog post might be relevant: https://blog.evantahler.com/productionizing-flynn-and-hosting-your-root-domain-s-fc849638e206#.58kmalw3d. Specifically the bits about updating **TLSPin** and **~/.gitconfig**.

**Comment by jxltom** [comment]
I have same issue ```tls internal error``` after migrate domain by ```flynn cluster migrate-domain```. I try to fix it by this post [Productionizing Flynn and Hosting your Root Domain(s)](https://blog.evantahler.com/productionizing-flynn-and-hosting-your-root-domain-s-fc849638e206#.58kmalw3d). It looks like I need to have a my own wildcard certificate for the new domain. But I don't have it.

So I have to remove flynn and reinstall it manually. By installing manually I can provide my custom domain during bootstrap without migrating after bootstrap.

This solves my issue temporally, but is there a solution for solving this tls internal error problem without re-installing?

**Comment by titanous** [comment]
I believe this was fixed by #3886.
